soc/silabs/silabs_siwx91x/siwg917/siwx91x_nwp.c

```diff
@@ -173,8 +187,10 @@ static void siwx91x_configure_sta_mode(sl_si91x_boot_configuration_t *boot_confi
 #ifdef CONFIG_WIFI_SILABS_SIWX91X
 	boot_config->ext_tcp_ip_feature_bit_map = SL_SI91X_CONFIG_FEAT_EXTENSION_VALID;
 	boot_config->ext_custom_feature_bit_map |=
-		SL_SI91X_EXT_FEAT_IEEE_80211W |
 		SL_SI91X_EXT_FEAT_FRONT_END_SWITCH_PINS_ULP_GPIO_4_5_0;
+	if (!IS_ENABLED(CONFIG_WIFI_SILABS_SIWX91X)) {
+		boot_config->ext_custom_feature_bit_map |= SL_SI91X_EXT_FEAT_IEEE_80211W;
+	}
```

Comment:
The `CONFIG_WIFI_SILABS_SIWX91X` is already tested by `#ifdef CONFIG_WIFI_SILABS_SIWX91X`, a few lines above.